core/src/main/java/org/mifos/mobilewallet/core/domain/model/Transaction.java

```diff
@@ -161,4 +161,142 @@ public String toString() {
                 ", receiptId=" + receiptId +
                 '}';
     }
+
+    private static String input;
+    private static String[] units=
```

Comment:
@shobhit24 Instead of creating the string array here, you should create the string array inside `strings.xml` file